core/src/main/java/com/linecorp/armeria/client/retry/RetryingClient.java

```diff
@@ -32,6 +32,7 @@
  */
 public abstract class RetryingClient<I extends Request, O extends Response>
         extends SimpleDecoratingClient<I, O> {
+    private static final int DEFAULT_MAX_ATTEMPTS = 5;
```

Comment:
I thought that this is just a safe mechanism. Most of clients will use the `Backoff` with `withMaxAttempts' when they need to set up the number of retry attempt.
This is for those who does not invoke that method. What do you think?